src/net_components/core_ops.jl

```diff
@@ -82,13 +74,17 @@ function tight_bound(
        bound_operator[bound_type](b_0, cutoff)
         return b_0
     end
-    relaxation = (tightening_algorithm == lp)
+    should_relax_binary_variables = (tightening_algorithm == lp)
     # x is not constant, and thus x must have an associated model
-    model = ConditionalJuMP.getmodel(x)
+    model = owner_model(x)
     @objective(model, bound_obj[bound_type], x)
```

Comment:
TODO: We need to fix the objective semantics here. Consider adding a test (although that seems challenging).